sdk/tables/Azure.Data.Tables/src/swagger/TableRestClient.cs

```diff
@@ -27,5 +28,91 @@ internal HttpMessage CreateDeleteRequest(string table, string requestId)
             request.Headers.Add("Accept", "application/json");
             return message;
         }
+
+        internal HttpMessage CreateQueryRequest(string requestId, QueryOptions queryOptions)
```

Comment:
Why did you have to add these?

Reply:
To add the Next*Key tokens to the query string in a consistent way. The swagger doesn't account for them.

Reply:
Should the swagger be fixed? With all the manual workarounds I like to know the path forwards.
Usually it's either:
1. Fixing swagger
2. Fixing generator
3. Keeping the workaround because it's to specific to be worth fixing anywhere else.

Reply:
In this case, I think the answer is to fix the swagger. I plan to comment the PR to request this change,